sdk/communication/communication-phone-numbers/test/utils/recordedClient.ts

```diff
@@ -54,7 +54,7 @@ export function createRecordedClient(
   const recorder = record(context, environmentSetup);
 
   return {
-    client: new PhoneNumbersClient(env.COMMUNICATION_CONNECTION_STRING),
+    client: new PhoneNumbersClient(env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING),
```

Comment:
We need to update the sample.env file as well, to use the new name for the variable